src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/LateBinding.vb

```diff
@@ -1014,7 +1061,14 @@ Namespace Microsoft.VisualBasic.CompilerServices
         End Function
 
         <DebuggerHiddenAttribute(), DebuggerStepThroughAttribute()>
-        Private Shared Function FastCall(ByVal o As Object, ByVal method As MethodBase, ByVal Parameters As ParameterInfo(), ByVal args As Object(), ByVal objType As Type, ByVal objIReflect As IReflect) As Object
+        Private Shared Function FastCall(
+                ByVal o As Object,
+                ByVal method As MethodBase,
+                ByVal Parameters As ParameterInfo(),
+                ByVal args As Object(),
+                ByVal objType As Type,
```

Comment:
>ByVal objType As Type [](http://example.com/codeflow?start=16&length=21)

Should this parameter be annotated with `<DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)>`?

Reply:
Not really required as the actual type that is being reflected on is `objIReflect` when we pass it down to `InvokeMemberOnIReflect` so just annotating that seems to be enough.